### PR TITLE
feat: add new method in billing api service

### PR DESCRIPTION
**Pull request by ms-fcabrera**
add new method in billing api service to get invoices

### CODE REVIEW

**Comment by jcamposmk** [comment]
can you add test to the service?

File: src/services/doppler-billing-user-api-client.double.ts [hunk 1]
```diff
@@ -108,6 +109,14 @@ export const fakeDeclinedInvoices = {
   totalPending: 8.0,
 };
 
+export const fakeInvoices = {
+  invoices: [
+    { date: new Date(), invoiceNumber: '123', amount: 5, error: '', status: 'declined' },
+    { date: new Date(), invoiceNumber: '123', amount: 5, error: '', status: 'approved' },
```
Comment (jcamposmk) [inline src/services/doppler-billing-user-api-client.double.ts:115]: Can invoices have different `numbers` and `amounts`?

Comment (ms-fcabrera) [reply src/services/doppler-billing-user-api-client.double.ts:115]: Yes

File: src/services/doppler-billing-user-api-client.test.ts [hunk 2]
```diff
@@ -386,4 +387,29 @@ describe('HttpDopplerBillingUserApiClient', () => {
...
+    // Assert
+    expect(request).toBeCalledTimes(1);
+    expect(result).not.toBe(undefined);
+    expect(result.success).toBe(true);
+    expect(request).toBeCalledWith(
+      expect.objectContaining({
+        method: 'GET',
+        url: '/accounts/email@mail.com/invoices?withStatus=pending&withStatus=approved',
+      }),
+    );
```
[the remark this answers is not in the document]
Comment (ms-fcabrera) [reply src/services/doppler-billing-user-api-client.test.ts:413]: expect.objectContaining() matches the object that has the same properties that i am defining, so it is a way to not have to write all the properties of the expected object. The other way to test the request could be
expect(request).toBeCalledWith({
    headers: {
       "Authorization": "bearer jwtToken",
    },
    method: "GET",
    url: "/accounts/email@mail.com/invoices?withStatus=pending&withStatus=approved",
}):
What do you think is best?

[the remark this answers is not in the document]
Comment (jcamposmk) [reply src/services/doppler-billing-user-api-client.test.ts:413]: I'm not sure, but, does something like this work? 

```
expect(request).objectContaining({
        method: 'GET',
        url: '/accounts/email@mail.com/invoices?withStatus=pending&withStatus=approved',
      });
```

[the remark this answers is not in the document]
Comment (ms-fcabrera) [reply src/services/doppler-billing-user-api-client.test.ts:413]: image

[the remark this answers is not in the document]
Comment (jcamposmk) [reply src/services/doppler-billing-user-api-client.test.ts:413]: No problem 💯  ...

**Comment by jcamposmk** [comment]
👍

**Pull request merged**